package/src/components/ChannelPreview/ChannelPreviewMutedStatus.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+
+import type { ChannelPreviewProps } from './ChannelPreview';
+
+import { useTheme } from '../../contexts/themeContext/ThemeContext';
+import { Mute } from '../../icons';
+import type {
+  DefaultAttachmentType,
+  DefaultChannelType,
+  DefaultCommandType,
+  DefaultEventType,
+  DefaultMessageType,
+  DefaultReactionType,
+  DefaultUserType,
+  UnknownType,
+} from '../../types/types';
+
+export type ChannelPreviewMutedStatusProps<
+  At extends UnknownType = DefaultAttachmentType,
+  Ch extends UnknownType = DefaultChannelType,
+  Co extends string = DefaultCommandType,
+  Ev extends UnknownType = DefaultEventType,
+  Me extends UnknownType = DefaultMessageType,
+  Re extends UnknownType = DefaultReactionType,
+  Us extends UnknownType = DefaultUserType,
+> = Pick<ChannelPreviewProps<At, Ch, Co, Ev, Me, Re, Us>, 'channel'> & {
+  muted: boolean;
+};
+
+/**
+ * This UI component displays an avatar for a particular channel.
+ */
+export const ChannelPreviewMutedStatus = <
+  At extends UnknownType = DefaultAttachmentType,
+  Ch extends UnknownType = DefaultChannelType,
+  Co extends string = DefaultCommandType,
+  Ev extends UnknownType = DefaultEventType,
+  Me extends UnknownType = DefaultMessageType,
+  Re extends UnknownType = DefaultReactionType,
+  Us extends UnknownType = DefaultUserType,
+>(
+  props: ChannelPreviewMutedStatusProps<At, Ch, Co, Ev, Me, Re, Us>,
+) => {
+  const { muted } = props;
+
+  const {
+    theme: {
+      colors: { grey_dark },
+    },
+  } = useTheme();
+
+  return muted ? (
+    <Mute height={20} pathFill={grey_dark} style={{ marginRight: 8 }} width={20} />
```

Comment:
Should we maybe set the width and marginRight here in the theme? Being able to change things by updating the theme tends to come up pretty frequently as a request

Reply:
Hey the style can be customizable using the theme but I doubt the height and width can be done that way as they don't apply by style, you need to pass it as a prop to the `RootSvg` component. Can we have a prop in that case here, `iconHeight` and `icon width`, or if you have a different solution for this? One more thing which might be possible is that we pass the style.height and style.width in the SVG itself by checking if we have those styles we can set them to respective height and weight prop. Not sure if this is the correct way, but we can solve this issue right away using this approach.